### ISSUE TITLE
CCActionFollow needs to provide access to calculated offset

### ISSUE THREAD

**Issue by Ben-G** [cocos2d/cocos2d-objc#828]
I ran into following issue: I want to use `CCActionFollow` to implement a camera. I have a scene where the node that should be followed is initially positioned off center. I blend into that scene with a transition and run `CCActionFollow` immediately. However, the action isn't run until the transition completed. As soon as the transition completes and `CCActionFollow` runs it causes the target node to jump.

 So I needed a way to set the position of the target node initially. For now I've added a category on `CCActionFollow` that uses the calculation from the `step` method and returns the resulting position:

```
- (CGPoint)currentOffset {
  if(_boundarySet)
  {
    // whole map fits inside a single screen, no need to modify the position - unless map boundaries are increased
    if(_boundaryFullyCovered)
      return [(CCNode *)_target position];

    CGPoint tempPos = ccpSub( _halfScreenSize, _followedNode.position);
    return ccp(clampf(tempPos.x, _leftBoundary, _rightBoundary), clampf(tempPos.y, _bottomBoundary, _topBoundary));
  }
  else
    return ccpSub( _halfScreenSize, _followedNode.position );
}
```

Basically there would be two ways to add this functionality to `CCActionFollow`:
- Add method as above to access calculated position
- Add a method that allows the developer to apply the calculated position immediately

Thoughts?
